src/HasteModuleLoader/__tests__/test_root/GlobalCSSStub.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule subdir3/GlobalCSSStub
```

Comment:
the reason this works is because of @providesModule. If you remove this line, the test will fail and that's what the task is about – to make that work :)